core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -51,7 +52,8 @@
      * Sets up request metrics.
      */
     public static void setup(RequestContext ctx, AttributeKey<Boolean> requestMetricsSetKey,
-                             MeterIdPrefixFunction meterIdPrefixFunction, boolean server) {
+                             MeterIdPrefixFunction meterIdPrefixFunction, boolean server,
+                             Function<? super RequestLog, Boolean> isSuccess) {
```

Comment:
`isSuccess` sounds like a boolean value. How about renaming it to `successFunction` and use `Predicate<? super RequestLog>`?
```suggestion
                             Predicate<? super RequestLog> successFunction) {
```